index.html

```diff
@@ -419,6 +419,454 @@ <h2>Capabilities Are Safer</h2>
+          other mitigation strategies here such as considering all zcaps
+          delegated from a particular controller as revoked or full key
+          revocation.
+        </p>
+
+        <p>
+          Delegated zcaps MUST have expiration date-times to support good
+          security hygiene practices and because zcaps support decentralized
+          delegation. In order to revoke a zcap, it must be submitted to the
+          verifier's revocation endpoint for the associated invocation target.
+          If a delegated zcap has been lost / misplaced, it MUST eventually
+          expire to avoid undesirable access.
+        </p>
+
+        <p>
+          A delegated zcap MAY have an allowedAction field that is a string or
+          an array of strings that each express an action that the controller of
+          the zcap may take when invoking the capability. A verifier MUST ensure
+          that the allowedAction field in a delegated zcap is not less
+          restrictive than the parent capability's, if present.
+        </p>
+
+        <p>
+          A delegated zcap MUST have a proof field that is an object or an
+          array of objects that each express a DI proof. At least one of these
+          proofs MUST be a zcap capability delegation proof
+          [TODO: more details on this proof].
+        </p>
+
+        <p>
+          A capability delegator (one who creates a delegated zcap) may
+          attenuate authority by setting a more restrictive expiration
+          date-time, a more restrictive invocation target (via URL path- or
+          query-based attenuation), or limiting the allowed actions. Taken
+          together, the API that a verifier manages access is expected to have
+          the flexibility required to model all desired authorization models.
+        </p>
+
+
+
+        <section id="delegated-capability-attenuation">
+          <h2>URL Path- or Query-based Attenuation</h2>
+
+          <p>
+            Remove @context / vocab-based caveats and replace with:
+          </p>
+
+          <p>
+            A verifier will accept delegations (and invocations) where a suffix
+            has been added to the parent zcap's invocation target (invoked zcap's
+            invocation target). The suffix MUST start with / or ? if the
+            invocation target prefix has no ? and & otherwise. This allows for
+            fully customizable attenuations via HTTP API path and query
+            parameters. For example, a ZCAP that can be invoked at
+            https://foo.example/bars/123 can be delegated with an attenuation
+            such that the delegated ZCAP has an invocation target of
+            https://foo.example/bars/123/bazzes/456. This could be further
+            delegated and attenuated with a ZCAP with an invocation target of
+            https://foo.example/bars/123/bazzes/456?day=tuesday and then again
+            with https://foo.example/bars/123/bazzes/456?day=tuesday&hour=12.
```

Comment:
```suggestion
            A verifier will accept delegations (and invocations) where a suffix
            has been added to the parent zcap's invocation target (invoked zcap's
            invocation target). The suffix MUST start with `/` or `?` if the
            invocation target prefix has no `?`, and with `&` otherwise. This allows for
            fully customizable attenuations via HTTP API path and query
            parameters. For example, a ZCAP that can be invoked at
            `https://foo.example/bars/123` can be delegated with an attenuation
            such that the delegated ZCAP has an invocation target of
            `https://foo.example/bars/123/bazzes/456`. This could be further
            delegated and attenuated with a ZCAP with an invocation target of
            `https://foo.example/bars/123/bazzes/456?day=tuesday` and then again
            with `https://foo.example/bars/123/bazzes/456?day=tuesday&hour=12`.
```